src/js/CommentPopup.js

```diff
@@ -0,0 +1,130 @@
+import updateTotalCommentsCount, { fetchMovieComments } from './commentCounter';
+
+const url1 = 'https://api.tvmaze.com/shows';
+const commentsEndpoint = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/DWcDAh13W8mhMfudWgwK/comments';
+const popup = document.querySelector('.movie-popup');
+
+const get = (url) => fetch(url)
+  .then((res) => res.json())
+  .then((data) => data)
+  .catch((error) => error);
+
+const post = (url, params = {}) => fetch(url, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: JSON.stringify(params),
+}).then((res) => res.text())
+  .then((data) => (data.error
+    ? { error: true, info: data }
+    : { error: false, info: data }))
+  .catch((error) => ({ error: true, info: error }));
+
+const addComment = async (params) => {
+  const response = await post(commentsEndpoint, params);
+  return response;
+};
+const fetchMovieData = async (movieId) => {
+  const response = await get(`${url1}/${movieId}`);
+  return response;
+};
+
+const displayMovieComments = (data) => {
+  popup.querySelector('.comments').innerHTML = data;
+};
+const enableClosePopup = () => {
+  document.querySelector('#close-popup').addEventListener('click', () => {
+    popup.style.display = 'none';
+    popup.innerHTML = '';
+  });
+};
+
+const showComments = (movieId) => {
+  fetchMovieComments(movieId).then((data) => {
+    if (!data.error) {
+      let comments = '';
+      data.forEach((comment) => {
+        comments += `<li>${comment.creation_date} ${comment.username}: ${comment.comment}</li>`;
+      });
+      displayMovieComments(comments);
+    } else {
+      displayMovieComments('No comments have been posted yet. Be the first to write something...');
+    }
+  });
+};
+const displayMoviePopup = (movieId) => {
+  popup.innerHTML = `Fetching data...<br>
+  <span id="close-popup">X</span>`;
+
+  fetchMovieData(movieId).then((data) => {
+    popup.innerHTML = `
+    <span id="close-popup">X</span>
+    <img src="${data.image.medium}" class="popup-img">
+    <h3 class="popup-title">${data.name}</h3>
+    <table>
+      <tr>
+        <td>
+          <b>premiered:</b> ${data.premiered}
+        </td>
+        <td>
+          <b>Ended:</b> ${data.ended}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <b>Language:</b> ${data.language}
+        </td>
+        <td>
+          <b>Type:</b> ${data.type}
+        </td>
+      </tr>
+    </table>
+    <h3 class="comment-title">
+    Comments
+    </h3>
```

Comment:
- [ ] As demonstrated in the guideline, there should be a count indicating the number of comments for the popup. Kindly implement that.

  Your Popup without count | Given popup with count
  :---:|:---:
  ![ezekiel-comments](https://user-images.githubusercontent.com/36542986/163363276-413e216d-ee51-4d74-99ee-a05630b03d61.jpg) | ![ezekiel-comm-design](https://user-images.githubusercontent.com/36542986/163363266-3da9dcce-d361-4a39-b524-14e1b79ed701.jpg)